Week-7/Problem 7.2 AS RotateLinkedList.js

```diff
@@ -0,0 +1,80 @@
+console.log("Rotate link list");
+console.log("Time complexity O(n) and space complexity O(1)");
+class SLL
+{
+    constructor(value)
+    {
+       
+        this.value=value;
+        this.next=null;
+    }
+}
+
+function createSLL(arr)
+{
+    let node = new SLL(arr[0]);
```

Comment:
can be const

Reply:
Crated a sperate file for classes and handle there.